src/Psalm/Internal/LanguageServer/LanguageServer.php

```diff
@@ -188,18 +188,14 @@ function (): void {
     /**
      * The initialize request is sent as the first request from the client to the server.
      *
-     * @param ClientCapabilities $capabilities The capabilities provided by the client (editor)
```

Comment:
Please revert changes here and just baseline issues. https://github.com/vimeo/psalm/pull/8960 plans to use those and make changes everywhere here, it will be easier if we don't change that before